db/seeds.rb

```diff
@@ -42,6 +44,34 @@ def describe_induction_period(ip)
   suffix = "(induction period)"
 
   print_seed_info("* is having their induction overseen by #{ip.appropriate_body.name} (AB) #{describe_period_duration(ip)} #{suffix}", indent: 4)
+
+  author_attributes = {
+    author_email: 'fkend@appropriate-body.org',
+    author_name: 'Felicity Kendall',
```

Comment:
❤️

Reply:
Can you believe @claire-hughez had never heard of Rosemary and Thyme? 🙄😂